src/system.h

```diff
@@ -63,6 +63,12 @@ GAP_STATIC_ASSERT(sizeof(void *) == SIZEOF_VOID_P, "sizeof(void *) is wrong");
 #endif
 #endif
 
+// If we are not running HPC-GAP, disable read and write guards.
+
+#ifndef HPCGAP
+#undef USE_HPC_GUARDS
```

Comment:
Why is this needed?

Reply:
I really don't like this. If this variable should always be undefined when not using HPCGAP, perhaps put in an #error if HPCGAP is undefined and USE_HPC_GUARDS is defined?

Reply:
We can remove this for now. It's left over from when the guards defaulted to enabled in the configure script. If we flip the default, we also have to turn this off, however.